wallet/tests/transaction.rs

```diff
@@ -57,8 +57,125 @@ fn setup(test_dir: &str, chain_dir: &str) -> Chain {
 	).unwrap()
 }
 
+/// Build and test new version of sending API
+#[test]
+fn build_transaction_2() {
```

Comment:
See this test fn for minimal transaction building workflow

Reply:
Looking nice!